crates/fj/build.rs

```diff
@@ -23,8 +23,8 @@ impl Version {
         let commit = git_description();
 
         let official_release =
-            std::env::var("FJ_OFFICIAL_RELEASE").as_deref() == Ok("1");
-        println!("cargo:rerun-if-env-changed=FJ_OFFICIAL_RELEASE");
+            std::env::var("release-detected").as_deref() == Ok("true");
```

Comment:
The name can be changed, as for me it is a bit misleading. I propose e.g. `is-official-release`.

Reply:
The release operator doesn't set any environment variables, it sets github actions outputs.
So if you want the `build.rs` to have access to the data, you need to export it in the step that will call `cargo build`.
At a glance, I'd say say this needs to happen here https://github.com/hannobraun/Fornjot/pull/1324/files#diff-ea3ea8c9932adc7ba8161ceda844fedd43b006848ef1140c050cbd7ea0788a18R65

Environment variables, as a best practice, are always written in all caps. So I would propose to call it `RELEASE_DETECTED` then.

On the aforementioned step - the one that runs `cargo build` - we could then populate the variable like so:

```yaml
- name: Some step
  env:
    RELEASE_DETECTED: needs.calculate-release-flags.outputs.release-detected
  # ...
```

Reply:
I'm open to renaming this output/variable, although I'm not sure about `is-official-release`. There's no such thing as an unofficial release, so what exactly does that mean? (Yes, I realize the previous environment variable did the same thing.) I agree with @hendrikmaus' point about capitalization.

Personally, I'd be fine with `RELEASE_DETECTED`, but that's just bikeshedding. I'd merge this pull request with any semi-reasonable naming choice. We can always improve things in follow-up PRs.